main.cpp

```diff
@@ -14,12 +14,11 @@ int main(int argc, char **argv) {
 
   std::ifstream input(argv[1]);
   xos::Lexer lexer(input);
-  // xos::Result<xos::Token> tok = lexer.Lex();
   xos::Parser parser(lexer);
   auto func = parser.parseFunc();
   xos::Compiler compiler;
   compiler.compile(*func);
-  compiler.TheModule->dump();
+  compiler.TheModule->print(llvm::outs(), nullptr);
```

Comment:
what's the difference? Can we not just dump?

Reply:
`dump` is only available for debug llvm builds, so prebuilt llvm libraries that are made in release mode (like the ones provided on the llvm release page) won't define this symbol and we'd run into an `undefined symbol` error when linking. `print` however is always available even in release builds. `dump` effectively calls `print` anyway.